pkg/antctl/command_definition.go

```diff
@@ -473,7 +496,7 @@ func (cd *commandDefinition) newCommandRunE(c *client) func(*cobra.Command, []st
 		if err != nil {
 			return err
 		}
-		return cd.output(resp, os.Stdout, formatterType(outputFormat), cd.getEndpoint().single() || argGet)
+		return cd.output(resp, os.Stdout, formatterType(outputFormat), cd.getEndpoint().OutputType() != multiple && (cd.getEndpoint().OutputType() == single || argGet))
```

Comment:
nit: it would be better to have a variable for `cd.getEndpoint().OutputType() != multiple && (cd.getEndpoint().OutputType() == single || argGet)`.

Reply:
Updated